lua/lualine.lua

```diff
@@ -34,7 +34,35 @@ local function load_components()
     for _, section in pairs(sections) do
       for index, component in pairs(section) do
         if type(component) == 'string' then
-          section[index] = require('lualine.components.' .. component)
+          local ok,loaded_component = pcall(require, 'lualine.components.' .. component)
+          if not ok then
+            -- vim veriable component
+            -- accepts g:, v:, t:, w:, b:, o, go:, vo:, to:, wo:, bo:
+            if component:find('[gvtwb]?o?:') == 1 then
```

Comment:
I would split this to handle both `g:something` and `vim.bo.something` to make it more distinct on when the vim syntax and lua syntax is used.

Reply:
vim.bo.something aren't currently handled . Actualy lua veriables csnn't be used as a component for now . Are you saying to implement just vim.b/vim.g kind of lua veriable and not other lua globals wonn't that be inconsistent?